tests/unit/dataactvalidator/test_b15_object_class_program_activity.py

```diff
@@ -11,8 +11,10 @@ def test_success(database):
     """ Tests that SF 133 amount sum for line 2104 matches the calculation from Appropriation based on the fields below
         for the specified fiscal year and period """
     tas = "".join([_TAS, "_success"])
+    tas2 = "".join([_TAS, "_other_tas"])
```

Comment:
For simple string concatenation, do we still want to use `join` or is `+` simple enough to use? I know it has to deal with flexibility/efficiency, but since this is just 2 simple strings, I'm curious if there's a difference.